debian/conf/override.ragnarok

```diff
@@ -0,0 +1,6 @@
+* Task Æsir
+odin Task Æsir
+odin Description-md5 654ca349ba8dade6580ba77c71d67eff
+thor Task Æsir
+thor Bugs https://bilskirnir.example.com/bugs
+odin Description-md5 654ca349ba8dade6580ba77c71d67eff
```

Comment:
should this be thor?

Reply:
Oh, nice catch, of course  :facepalm:

Reply:
And still i have no idea what it's for...